src/cephadm/tox.ini

```
@@ -65,3 +65,13 @@ commands =
     bash -c "test $(grep -c 'docker.io' cephadm.py) == 11"
 # Downstream distributions may choose to alter this "docker.io" number,
 # to make sure no new references to docker.io are creeping in unnoticed.
+
+# coverage env is intentionally left out of the envlist. It is here for developers
+# to run locally to generate and review test coverage of cephadm.
+[testenv:coverage]
+skip_install=true
+deps =
+  {[testenv]deps}
+  pytest-cov
+commands =
+  pytest -v --cov=cephadm --cov-report=html {posargs}
```

```suggestion
  pytest -v --cov=cephadm --cov-report=term-missing --cov-report=html {posargs}
```

(nit) can we also add a `term` or `term-missing` report? otherwise this lgtm!

---

Sure. Done!